package.json

```diff
@@ -22,7 +22,7 @@
     "build": "gulp build",
     "coverage": "gulp check",
     "dev": "gulp dev",
-    "start": "gulp server",
+    "start": "gulp build && gulp server",
```

Comment:
This is causing the issue in Travis. Basically, there is no time for the server to start up since it is waiting for the build. I think `npm run start` should only start up the server. Maybe move these commands to a different npm script, like `build-start` or something similar.